e2e/smoke_test.go

```diff
@@ -201,16 +201,15 @@ var _ = Describe("Smoke End To End Tests - against OpenShift Cluster with Istio
 						EnsureAllPodsAreReady(namespace)
 						EnsureProdRouteIsReachable(namespace, ContainSubstring("ratings-v1"))
 
-						ike := RunIke(tmpDir, "develop",
+						ike := RunIke("../", "develop",
```

Comment:
You can also use `GetProjectDir()`

https://github.com/maistra/istio-workspace/blob/8fe3f8e92cb0cb74605f74d8fc582cd67c4ef853/test/shell/funcs.go#L44-L50